docs/ops/generation/RandomUniform_8.md

```diff
@@ -0,0 +1,246 @@
+## RandomUniform <a name="RandomUniform"></a> {#openvino_docs_ops_generation_RandomUniform_8}
+
+**Versioned name**: *RandomUniform-8*
+
+**Category**: Generation
+
+**Short description**: *RandomUniform* operation generates a sequence of random values from a uniform distribution.
+
+**Detailed description**:
+
+*RandomUniform* operation generates random numbers from a uniform distribution in the range [*minval*, *maxval*). The generation algorithm is based on underlying random integer generator that uses Philox algorithm. Philox algorithm is counter based pseudo random generator, which produces unit32 values. Single invocation of Philox algorithm returns four result random values, depending on the given *key* and *counter* values. *Key* and *counter* are initialized with *seed* and *seed2* attributes respectively.
+
+\f[
+key = seed\\
+counter = seed2
+\f]
+
+Link to original paper: https://www.thesalmons.org/john/random123/papers/random123sc11.pdf
+
+The result of Philox is calculated by applying a fixed number of *key* and *counter* updating, so-called "rounds". This implementation uses 4x32_10 version of Philox algorithm, where number of rounds = 10.
+
+Suppose we have *n* which determines *n*-th 4 elements of random sequence.
+In each round *key*, *counter* and *n* are splitted to pairs of uint32 values:
+
+\f[
+R = cast\_to\_uint32(value)\\
+L = cast\_to\_uint32(value >> 32),
+\f]
+where *cast\_to\_uint32* - static cast to uint32, *value* - uint64 input value, *L*, *R* - uint32 result values, >> - bitwise right shift.
+
+Then *n* and *counter* are updated with the following formula:
+
+\f[
+L'= mullo(R, M)\\
+R' = mulhi(R, M) {\oplus} k {\oplus} L \\
+mulhi(a, b) = floor((a {\times} b) / 2^{32}) \\
+mullo(a, b) = (a {\times} b) \mod 2^{32}
+\f]
+where *k* = `R_{key}` for updating counter, *k* = `L_{key}` for updating *n*, *M* = `0xD2511F53` for updating *n*, *M* = `0xCD9E8D57` for updating *counter*.
```

Comment:
Is there a way to insert formula in text inline, for the correct display of `R_{key}` and `L_{key}` symbols?

Reply:
@andrew-zaytsev, could you please help with advice: "Is there a way to insert formula in text inline, for the correct display of R_{key} and L_{key} symbols?"